test/widgets/actions_test.dart

```diff
@@ -52,8 +52,9 @@ void main() {
         processedCount: 11, updatedCount: 3,
```

Comment:
There is one more call-site of `pumpAndSettle` from `prepare` that can be cut too.